clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -125,6 +126,17 @@ public UUIDSerde() {
         }
     }
 
+    static public final class ListSerde<Inner> extends WrapperSerde<List<Inner>> {
+
+        public ListSerde() {
+            super(new ListSerializer<>(), new ListDeserializer<>());
+        }
+
+        public <L extends List> ListSerde(Class<L> listClass, Serde<Inner> serde) {
```

Comment:
should be `<L extends List<Inner>>` to avoid raw type warning and make build pass

Reply:
Same as below -- I guess my suggestion does not work in practice (it's correct that it would avoid the raw type warning, but it would make the API unusable in practise because we want to be able to pass in raw type list classes).